### ISSUE TITLE
The NRC Emotion Intensity Lexicon (NRC-EIL)

### ISSUE THREAD

**Issue by LinguList** [concepticon/norare-data#124]
http://www.saifmohammad.com/WebPages/AffectIntensity.htm

They offer also some 100 translations of their words, the website is generally interesting, as they also say that there seems to be quite some correlation across langauges, which again shows that words are a good proxy for concepts (on large scale).

**Comment by AnnikaTjuka** [comment]
Yes, we have the list ([Mohammad-2018-EmotionIntensity](https://github.com/concepticon/norare-data/tree/master/concept_set_meta/Mohammad-2018-EmotionIntensity)) for English. The translations were in a separate list.

**Comment by LinguList** [comment]
Okay, then we should check out how many translations we may add.
